WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/TemplatePreviewViewController.swift

```diff
@@ -100,10 +100,13 @@ class TemplatePreviewViewController: UIViewController, NoResultsViewHost, UIPopo
     }
 
     private func styleButtons() {
+        let isLastSiteCreationStep = ABTest.siteNameV1.variation == .treatment(nil) && FeatureFlag.siteName.enabled
+        let mainButtonTitle = isLastSiteCreationStep ? NSLocalizedString("Create site", comment: "Title for the button to progress with creating the site with the selected design")
```

Comment:
since we have a couple of hardcoded strings, including this one, would it be possible to group them in a private `enum`?

Reply:
Great suggestion @Gio2018 👍 
Handled with babc542445c63b38afa12643abe19746e3dc30cb